ast/builtins.go

```diff
@@ -1715,6 +1716,19 @@ var JSONIsValid = &Builtin{
 	Categories: encoding,
 }
 
+var RenderTemplate = &Builtin{
+	Name:        "render_template",
+	Description: "Renders a templated string with given template variables injected. See https://pkg.go.dev/text/template for templating examples.",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("x", types.S).Description("a string"),
```

Comment:
Nit: Can we elaborate the desc a bit. For example `a templated string` or something.